### PR TITLE
Fix CWE-117

### PR DESCRIPTION
**Pull request by tdrozdovsky**
Signed-off-by: Taras Drozdovskyi <t.drozdovsky@samsung.com>

# Description

Fixed 4 LGTM alerts and marked 2 LGTM alerts as false positive

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Documentation update

# How Has This Been Tested?

After applied this PR, the LGTM status will be updated

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: internal/controller/securemgr/verifier/verifier.go [hunk 1]
```diff
@@ -152,7 +153,7 @@ func addHashToContainerWhiteList(hash string) error {
 	fileContent, err := ioutil.ReadFile(cwlFilePath)
 	if err != nil {
 		fileContentStr := hash + "\n"
-		err = ioutil.WriteFile(cwlFilePath, []byte(fileContentStr), 0666)
+		err = ioutil.WriteFile(cwlFilePath, []byte(fileContentStr), 0666) // lgtm [go/path-injection]
```
Comment (t25kim) [inline internal/controller/securemgr/verifier/verifier.go:156]: I think this is not CWE-117 and the `hash` should be verified in the previous [function](https://github.com/lf-edge/edge-home-orchestration-go/blob/c5e39bd14ab104f9f9b84fd0a1b7c1874e69ceca/internal/restinterface/externalhandler/externalhandler.go#L337) in advance.
What do you think?

File: internal/controller/securemgr/verifier/verifier.go [hunk 2]
```diff
@@ -167,12 +168,12 @@ func addHashToContainerWhiteList(hash string) error {
...
 		for _, whitelistItem := range containerWhiteList {
 			if whitelistItem == hash {
-				log.Printf("%s container's hash %s already exists in container white list\n", logPrefix, hash)
+				log.Info(logPrefix, " container's hash ", logmgr.SanitizeUserInput(hash), " already exists in container white list") // lgtm [go/log-injection]
 				return nil
 			}
 		}
 		fileContentStr = fileContentStr + hash + "\n"
-		err = ioutil.WriteFile(cwlFilePath, []byte(fileContentStr), 0666)
+		err = ioutil.WriteFile(cwlFilePath, []byte(fileContentStr), 0666) // lgtm [go/path-injection]
```
Comment (t25kim) [inline internal/controller/securemgr/verifier/verifier.go:176]: It is the same as above.

**Review by t25kim — COMMENTED** [review]
Thanks for your contribution!! PTAL

File: internal/controller/securemgr/verifier/verifier.go (hunk 1 above)
Comment (tdrozdovsky) [reply internal/controller/securemgr/verifier/verifier.go]: @t25kim Thank you for your opinion. This remark does not apply to the CWE-117 (as I mentioned in the text of the PR). **I think your idea is great.** I have already made a draft version and after testing I will create a separate PR.

**Review by t25kim — APPROVED** [review]
LGTM! Thanks for your steady contribution!

**Review by MoonkiHong — APPROVED** [review]
+1

**Pull request merged**